### PR TITLE
Force port mapping

### LINKED ISSUE #2: Being able to override port in pomdok configuration
In some case, we need to have a fixed port number for our applications, we can override it with Symfony CLI: `symfony local:server:start --port=8000`

### PR DESCRIPTION
**Pull request by Korbeil**
Fixes #2 

Will allow to force port in your configuration file as following :point_down: 
```yaml
pomdok:
  tld: 'test'
  projects:
    - domain: 'api.project'
      path: '/apps/api'
      port: 9990
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,6 +39,8 @@ pomdok:
 
 You'll need at least `tld` field and one project to have a valid configuration.
 
+For each "project" you have, you'll need atleast a `domain` and `port` fields. `port` field is optional and used to force a given port for your webserver.
```
Comment (pyrech) [inline README.md]:
```suggestion
For each "project" you have, you'll need at least a `domain` and `port` fields. `port` field is optional and used to force a given port for your webserver.
```

File: startAndStopCommand.go [hunk 2]
```diff
@@ -58,12 +58,18 @@ func startOrStopCommand(command string, message string) {
...
-		fmt.Printf("%s %s\n", message, yellow(fmt.Sprintf("%s.%s", domain, symfonyJsonData.Tld)))
+	symfonyJSONData := SymfonyJSONProxy{}
+	json.Unmarshal(file, &symfonyJSONData)
+
+	for domain, path := range symfonyJSONData.Domains {
+		forcedPort := symfonyJSONData.Ports[domain]
+		formattedCommand := fmt.Sprintf("/usr/local/bin/symfony %s --dir=%s", command, path)
+		if "local:server:start --daemon" == command && 0 != forcedPort {
+			formattedCommand = fmt.Sprintf("/usr/local/bin/symfony %s --port=%d --dir=%s", command, forcedPort, path)
+		}
```
Comment (matthieuwerner) [inline startAndStopCommand.go:70]: A lot of "inline things", is it possible to improve this part ?

File: initCommand.go [hunk 3]
```diff
@@ -27,14 +28,7 @@ var initCommand = &cli.Command{
...
 		argv := ctx.Argv().(*initT)
-		file := findFileUp(argv.Config, 0)
-		if file == "" {
-			return nil
-		}
-
-		data, _ := ioutil.ReadFile(file)
-		config := PomdokYamlConfig{}
-		yaml.Unmarshal([]byte(data), &config)
+		config, baseDirectory, _ := loadPomdokConfig(argv.Config)
```
Comment (qboot) [inline initCommand.go]: Why bubble an error if we don't handle it here ?

**Review by qboot — COMMENTED** [review]
Sounds good to me 👍

File: startAndStopCommand.go [hunk 4]
```diff
@@ -58,12 +58,19 @@ func startOrStopCommand(command string, message string) {
...
-	json.Unmarshal(file, &symfonyJsonData)
+	symfonyJSONData := SymfonyJSONProxy{}
+	json.Unmarshal(file, &symfonyJSONData)
 
-	for domain, path := range symfonyJsonData.Domains {
-		runCommand(fmt.Sprintf("/usr/local/bin/symfony %s --dir=%s", command, path))
-		fmt.Printf("%s %s\n", message, yellow(fmt.Sprintf("%s.%s", domain, symfonyJsonData.Tld)))
+	for domain, path := range symfonyJSONData.Domains {
+		forcedPort := symfonyJSONData.Ports[domain]
+		formattedCommand := fmt.Sprintf("/usr/local/bin/symfony %s --dir=%s", command, path)
```
Comment (lyrixx) [inline startAndStopCommand.go]: Why do you harcode the path?

**Pull request merged**